app/experimenter/experiments/migrations/0081_remove_experiment_risk_internal_only.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 3.0.2 on 2020-02-05 23:53
+
+from django.db import migrations
+from django.db.models import Q
+
+
+def update_risk_confidential(apps, schema_editor):
+    Experiment = apps.get_model("experiments", "Experiment")
+    experiments = Experiment.objects.filter(
+        Q(risk_confidential=True) | Q(risk_internal_only=True)
+    )
+    experiments.update(**{"risk_confidential": True})
```

Comment:
Isn't this superfluous?  Can't you just say `.update(risk_confidential=True)`?

Reply:
And you can tack the `.update` onto the end of the `.filter` and don't need the variable at all?